chingu-voyages/v50-tier2-team-14
# Feature/Connect map to menu

# Description
 -Connected created map  to menu. now when a user clicks on  a marker on the map they will be redirected to that restaurant menu page
- updated routes to include menu and cart pages
- added redirection to home page when clicking on logo on the navbar
- added redirection to cart when clicking on the cart icon in the navbar


## How to test:
1. check into current branch
2. do `npm install` and `npm start` to run this PR locally
3. go to `localhost:3000`
4. from the main page click on the cart icon , verify that you see the cart
5. form the cart page click on the logo, verify that you are taken to homepage
6. on the map, select a restaurant and click on it, verify that you are taken to that restaurant's menu page
